.github/workflows/stage_call.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
           python-version: "3.12"
           cache: "pip" # caching pip dependencies
       - run: pip install -r requirements.txt
-      - name: stage
```

Comment:
Is the switch from `name` to `id` here accidental from our recent refactor of "model-name" to "resource-id"?

Reply:
no, intentional as its outputs are used

Reply:
(and while chaning I thought the name is just hiding the python script call)